tikv-configuration-file.md

```diff
@@ -370,9 +370,13 @@ Configuration items related to storage.
 ### `enable-ttl`
 
 + TTL is short for "Time to live". If this item is enabled, TiKV automatically deletes data that reaches its TTL. To set the value of TTL, you need to specify it in the requests when writing data via the client. If the TTL is not specified, it means that TiKV does not automatically delete the corresponding data.
-+ Note: The TTL feature is only available for the RawKV interface for now. You can only configure this feature when creating a new cluster because TTL uses different data formats in the storage layer. If you modify this item on an existing cluster, TiKV reports errors when it starts.
 + Default value: `false`
 
+> **Warning:**
+>
+> - The TTL feature is only available for the RawKV interface for now. You can only configure this feature when creating a new cluster because TTL uses different data formats in the storage layer.
+> - You **must not** modify this item on an existing cluster. Otherwise, TiKV reports errors when it starts.
+> - You **must not** use this item on a cluster with TiDB nodes. Otherwise, serious consequences might occur, such as data corruption and the upgrade failure of TiDB clusters.
```

Comment:
```suggestion
> - You **must not** use this item on a cluster with TiDB nodes. Otherwise, serious consequences might occur, such as data corruption and the upgrade failure of TiDB clusters.

```